src/core/mount-setup.c

```diff
@@ -405,7 +406,15 @@ int mount_setup(bool loaded_policy) {
          * systemd. */
         mkdir_label("/run/systemd", 0755);
         mkdir_label("/run/systemd/system", 0755);
-        mkdir_label("/run/systemd/inaccessible", 0000);
+        mkdir_label(SD_INACCESSIBLE_PATH, 0755);
```

Comment:
I think this one should also be 0000. Not that there was a strong reason for this, but there's also no reason to allow normal users to peek into this, given that it does contain some weird stuff...